models/index.js

```diff
@@ -5,12 +5,14 @@ const path = require('path');
 const Sequelize = require('sequelize');
 const process = require('process');
 const basename = path.basename(__filename);
+require('dotenv').config();
 const env = process.env.NODE_ENV || 'development';
 const config = require(__dirname + './../config/database.js')[env];
 const db = {};
+const { DB_USER, DB_PASSWORD, DB_HOST,DB_DATABASE } = process.env;
 
 const sslOptions =
-  env === 'development'
+  env === 'development '
```

Comment:
I don't think this is a good solution. Can you instead change the dev script to run just nodemon in package.json to 

```
 "dev": "npx nodemon index.js"
 ```